extensions/resource-deployment/package.json

```diff
@@ -13,16 +13,16 @@
     "azdata": "^1.6.0"
   },
   "activationEvents": [
-    "onCommand:azdata.resource.sql-image.deploy",
-    "onCommand:azdata.resource.sql-bdc.deploy"
+    "*"
```

Comment:
i had to change this to * due to https://github.com/microsoft/azuredatastudio/issues/5841